index.js

```diff
@@ -26,27 +26,31 @@ function handler() {
     render(state);
 }
 
+//first case with SetTimeout
+
+let seconds = 0;
+
 function update() {
+    if(seconds >= 60) {
+        return;
+    }
```

Comment:
your solution works. good.

but: can you adapt it to use `clearTimeout`, and without counting seconds? 
https://developer.mozilla.org/en-US/docs/Web/API/WindowOrWorkerGlobalScope/clearTimeout

hint: you'll need another `setTimeout`